.github/workflows/deploy-to-obs-landing.yaml

```diff
@@ -0,0 +1,55 @@
+name: deploy-to-obs-landing
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build-obs-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup SSH Client
+        uses: webfactory/ssh-agent@v0.5.4
+        with:
+          ssh-private-key: ${{ secrets.DEPLOY_KEY_PRIVATE }}
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ssh-key: ${{ secrets.DEPLOY_KEY_PRIVATE }}
+      - name: Checkout obs-landing repo
+        uses: actions/checkout@v3
+        with:
+          repository: openSUSE/obs-landing
+          path: obs-landing
+          ssh-key: ${{ secrets.DEPLOY_KEY_PRIVATE }}
+      - name: build
+        uses: addnab/docker-run-action@v3
+        with:
+          image: openbuildservice/obs-docu:latest
+          options: -v ${{ github.workspace }}:/github/workspace/
+          run: |
+            pushd /github/workspace/
+            for doc in obs-admin-guide obs-user-guide; do
+              ROOTID=${doc%-guide}
+              source ./DC-$doc || exit 1
+
+              # update html
+              daps --verbosity=3 html || exit 1
+              daps --verbosity=3 epub || exit 1
```

Comment:
According to our statistics, EPUB isn't really used anymore. So maybe it would be a good opportunity to just don't offer this format?

Reply:
@hennevogel can we remove the EPUB format?

Reply:
@tomschr even its no in heavily used anymore, we thought it doesn't really hurt to still provide the EPUB version

Reply:
It's up to you, of course. :slightly_smiling_face: Just wanted to tell you, it's not very popular among our readers. :wink: